### PR TITLE
Make tests pass in Julia 0.6.

### PR DESCRIPTION
**Pull request by maxvoxel8**

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,3 +1,3 @@
 julia 0.4
-DataStructures 0.3.8
+DataStructures 0.5.3
```
Comment (tkelman) [inline REQUIRE]: this should be 0.5.0 https://github.com/JuliaCollections/DataStructures.jl/compare/v0.4.6...v0.5.0

File: src/a_star_spath.jl [hunk 2]
```diff
@@ -30,8 +31,16 @@ function a_star_impl!{V,D}(
 
     tindx = mkindx(t)
 
+    if VERSION > v"0.6-"
```
Comment (tkelman) [inline src/a_star_spath.jl]: should probably always use DataStructures, I believe it's the same code

File: REQUIRE (hunk 1 above)
Comment (dehann) [inline REQUIRE]: Will do thanks

File: REQUIRE [hunk 3]
```diff
@@ -1,3 +1,3 @@
 julia 0.4
```
Comment (tkelman) [inline REQUIRE]: dropping 0.4 is still a good idea

**Pull request merged**